src/App.jsx

```diff
@@ -14,70 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+function isSelected(value, good) {
+  return value === `${good} is selected`;
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [value, setValue] = useState('Jam is selected');
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {value}
+        {
+          value !== 'No goods selected'
+          && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => {
+                setValue('No goods selected');
+              }}
+            />
+          )
+        }
+      </h1>
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={
+                isSelected(value, good) ? 'has-background-success-light' : ''
+              }
             >
```

Comment:
Don't forget to add `key` when rendering
```suggestion
          {goods.map(good => (
            <tr
              data-cy="Good"
              className={
                isSelected(value, good) ? 'has-background-success-light' : ''
              }
            >
```